packages/nouvelle/README.md

```diff
@@ -2,8 +2,120 @@
 
 The Kendo UI Nouvelle Theme is a css variables heavy theme for the Kendo UI components.
 
+* [Quick start](#quick-start)
+* [Customizing](#customizing)
+* [Integrating with third party frameworks](#integrating-with-third-party-frameworks)
+* [Bug reports and feature requests](#bug-reports-and-feature-requests)
+* [Versioning](#versioning)
 * [License](#license)
-* [Basic Usage](#basic-usage)
+
+## Quick start
+
+Kendo UI Nouvelle Theme is available as an npm package.
+
+If you don't need to apply any [customization](#customizing) to the theme, you can can reference the precompiled CSS directly from unpkg CDN:
+
+```html
+<link rel="stylesheet" href="https://unpkg.com/@progress/kendo-theme-nouvelle/dist/all.css" />
+```
+
+While using the precompiled CSS file is faster than compiling the theme from the source code, the approach has the two drawbacks: it includes CSS for all components; it does not provide options for theme customization through SCSS variables (which is possible when you build the theme from the source code) because the theme is already compiled.
+
+If you prefer, or you need to customize the theme, you can install it:
+
+```sh
+npm install --save @progress/kendo-theme-nouvelle
+```
+
+and then import it in your project styles:
+
+```scss
+// Import the entire theme
+@import "~@progress/kendo-theme-nouvelle/dist/all.scss";
+```
+
+For more information on how to implement the Nouvelle theme in your project, refer to the following articles:
+
+* [Using the Nouvelle Theme in Angular Projects](https://www.telerik.com/kendo-angular-ui/components/styling/theme-nouvelle/)
```

Comment:
We don't have such pages on this URL yet.

Reply:
no, we dont, but I have to link to somewhere :D I am guessing, we should either have a generic page for that, or such pages for all themes. Also, we can track how many people click on the links

Reply:
I'll go ahead and create such pages for the KendoReact, Kendo UI for Angular and Kendo UI for Vue docs sites: https://github.com/telerik/katana/pull/1098